lms/envs/common.py

```diff
@@ -1291,25 +1289,25 @@
         ],
         'output_filename': 'css/lms-style-course-vendor.css',
     },
-    'style-student-notes': {
-        'source_filenames': [
-            'css/vendor/edxnotes/annotator.min.css',
-        ],
-        'output_filename': 'css/lms-style-student-notes.css',
-    },
     'style-course': {
         'source_filenames': [
-            'sass/course.css',
+            'sass/lms-course.css',
             'xmodule/modules.css',
         ],
-        'output_filename': 'css/lms-style-course.css',
+        'output_filename': 'css/lms-course.css',
```

Comment:
@feanil, will this have any cache/production repercussions when this gets merged and a release is cut?